integration/run_test.go

```diff
@@ -110,6 +115,12 @@ func TestRunMultipleContainers(t *testing.T) {
 				sh.X("soci", "run", "-d", "--rm", "--snapshotter=soci", image, getTestContainerName(index, container))
 			}
 
+			// Verify that no mounts fallback to overlayfs
+			curlOutput := string(sh.O("curl", tcpMetricsAddress+metricsPath))
```

Comment:
Why shell out to curl instead of using net/http or a higher level request library?

Reply:
"localhost" is in the context of the testing container. Using net/http would still require us to build some kind of binary inside the container and run it.